Doc/whatsnew/3.12.rst

```diff
@@ -282,6 +282,12 @@ Removed
   a C implementation of :func:`~hashlib.pbkdf2_hmac()` which is faster.
   (Contributed by Victor Stinner in :gh:`94199`.)
 
+* :mod:`xml.etree`: Remove the ``ElementTree.Element.copy()`` method of the
```

Comment:
Oh right, I forgot about that. Updated PR. I was confused that the change only impacts the pure Python implementation. I guess that nobody will notice the change, since the C accelerator is used by default.